.travis.yml

```diff
@@ -1,16 +1,26 @@
+# Use the default infrastructure (Container-based Ubuntu 14.04)
```

Comment:
This comment will be stale when Travis updates Ubuntu. It's not necessary, and should be removed. Those working with Travis know what `sudo: false` does. Those new to Travis will learn pretty quickly without this comment.